### PR TITLE
October 2019

### PR DESCRIPTION
**Pull request by CaptV0rt3x**

### CODE REVIEW

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      "./RE 4.png"
+      "./RE 4 II.png"
+    >}}
+
+### GPU Emulation
+
+`Public Changes`
+
+- [#2891](https://github.com/yuzu-emu/yuzu/pull/2891) - Implemented `RGBX16F Format` which is used by **Rocket League** to render Graphics. 
+- [#2870](https://github.com/yuzu-emu/yuzu/pull/2870) - Implemented an `MME Draw calls inliner` which unifies instances draw calls into many a single draw call, improving performance significantly in some titles.
```
Comment (Schplee) [inline site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: `which unifies instances draw calls into many a single draw call`
Wording is awkward. Can you confirm with the original developer that the terminology is correct? 

Should it be "unifies multiple instances of draw calls into a single draw call"? No idea, can't tell by reading.

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+### GPU Emulation
+
+`Public Changes`
+
+- [#2891](https://github.com/yuzu-emu/yuzu/pull/2891) - Implemented `RGBX16F Format` which is used by **Rocket League** to render Graphics. 
+- [#2870](https://github.com/yuzu-emu/yuzu/pull/2870) - Implemented an `MME Draw calls inliner` which unifies instances draw calls into many a single draw call, improving performance significantly in some titles. 
+- [#2869](https://github.com/yuzu-emu/yuzu/pull/2869), [#2878](https://github.com/yuzu-emu/yuzu/pull/2878), and [#2855](https://github.com/yuzu-emu/yuzu/pull/2855) - Implemented shader instructions: `SULD`, `ICMP`, `SHFL`(Nvidia only) and `SUATOM`.
+
+- `Fix` - Fixed Memory mapping so that it won't call GPU flushing unnecessarily anymore.
```
Comment (Schplee) [inline site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: Memory and Stencil are capitalized, but not mipmap? Why are they arbitrarily capitalized? If you're going to capitalize things for emphasis, it should be consistent across the board

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    "./dungeon_maker.png"
+    "./dungeon_maker2.png"
+>}}
+
+## Et al.
+
+Please remember that the early access features are still being worked on, so if you encounter any bugs or have any feedback, please don't hesitate to share it with us in our patreon channels on our [Discord server](https://discord.gg/u77vRWY).
+
+As always, we are thankful to all our 450+ patrons for their continued support.
+We will keep working diligently to bring you many more improvements, so that you can keep enjoying your favourite games on yuzu.
```
Comment (Schplee) [inline site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: Just a nit, `favourite` purposefully spelled that way? Ignore if you don't want to use the american "favorite" :P

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+- [#2868](https://github.com/yuzu-emu/yuzu/pull/2868) - Fixed mipmap filtering. 
+- [#2833](https://github.com/yuzu-emu/yuzu/pull/2833) - Fixed Stencil testing. 
+- [#2914](https://github.com/yuzu-emu/yuzu/pull/2914) - Corrected `Fermi2D` to work within crop areas in order to avoid issues in Vulkan.
+- [#2917](https://github.com/yuzu-emu/yuzu/pull/2917) - Added deduction method for `blits` which helps figure if a `blit` is on depth textures. 
+This avoids using the fall-back method of buffer copy.
+&nbsp;
+
+`Patreon Changes`
+
+- Implemented `fast BRX` (aka evil shaders) and fixed `TXQ`.
```
Comment (FearlessTobi) [inline site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: As a user I would figure from this that we now implemented evil shaders, whatever that would be. Maybe change it to `fixes evil shaders` at least?

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+`Public Changes`
+
+- [#2877](https://github.com/yuzu-emu/yuzu/pull/2877) - Implemented `REV5 Audio`'s frame count.
+- [#2873](https://github.com/yuzu-emu/yuzu/pull/2873) - Implemented `ioctl2` & `ioctl3` interfaces in `NVDRV` (Nvidia Services).
+- [#2912](https://github.com/yuzu-emu/yuzu/pull/2912) - Fixes and Improvements to Asynchronous GPU:
+  
+  * New games boot with `Async` - Fire Emblem: 3 Houses, Cadence of Hyrule, and Pokken Tournament.
+  * Fixed the issue where games would suddenly slowdown and report wrong framerates.
+  * Fixed the issue where some games would surpass frame limiting.
+  * Fixed the issue where dynamic resolution re-scaler would trigger in **Super Mario Odyssey** with full-speed and won't trigger heavily for under-speed games.
```
Comment (FearlessTobi) [inline site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: not sure if it's just me, but shouldn't both cases either use would or simple future?

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
++++
+date = "2019-10-07T23:48:00+05:30"
+title = "yuzu Patreon Release October 2019"
+author = "CaptV0rt3x"
+forum = 150506
++++ 
+
+Heya ***yuz-ers***!
```
[the remark this answers is not in the document]
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: done

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+Wanna know what we have in store for you?
+The October Patreon preview release!
+This build showcases the new game in town, **The Legend of Zelda: Link's Awakening**!
+Hop right in, to know more.
```
[the remark this answers is not in the document]
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: done

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 1 above)
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: done

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 2 above)
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: done

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 5 above)
Comment (Schplee) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: I believe it should be "wouldn't" if I understand the sentence correctly, but it does seem awkwardly phrased

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 3 above)
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: ignored :P

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 4 above)
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: removed that reference

File: site/content/entry/yuzu-patreon-preview-oct-2019/index.md (hunk 5 above)
Comment (CaptV0rt3x) [reply site/content/entry/yuzu-patreon-preview-oct-2019/index.md]: fixed

**Pull request closed without merging**